### DISCUSSION TITLE
133

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#65] [General]
# 潮流周刊第133期 - 树上的鸟

封面来源于上周随手拍的一个在树上叼着木条的鸟，叽叽喳喳，让原本静止的树有一点儿生机。

https://weekly.tw93.fun/posts/133-%E6%A0%91%E4%B8%8A%E7%9A%84%E9%B8%9F

**Comment by nanvon** [comment]
“Flash 保存计划” 的图片放错了，和上面重复啦

**Reply by tw93** [reply]
感谢提醒，已经修正
